software/earl/authenticator_test.go

```diff
@@ -222,12 +223,17 @@ func TestTimeLimits(t *testing.T) {
 		defer syscall.Unlink(authFile.Name())
 	}
 
-	someMidnight, _ := time.Parse("2006-01-02", "2014-10-10") // midnight
-	nightTime_3h := someMidnight.Add(3 * time.Hour)           // 03:00
-	earlyMorning_7h := someMidnight.Add(7 * time.Hour)        // 09:00
-	hackerDaytime_13h := someMidnight.Add(13 * time.Hour)     // 16:00
-	closingTime_22h := someMidnight.Add(22 * time.Hour)       // 22:00
-	lateStayUsers_23h := someMidnight.Add(23 * time.Hour)     // 23:00
+	now := time.Now()
```

Comment:
This time should be a fixed time that you choose. Whatever, 4. May of 2015, because then you can do exact comparisons if something is within the time.